kotlin-testing/build.gradle.kts

```diff
@@ -0,0 +1,27 @@
+plugins {
+    kotlin("jvm")
```

Comment:
Shouldn't we call the module kotlin-jvm-testing` then?

Reply:
That's neither better nor worse